extensions/ql-vscode/src/databases.ts

```diff
@@ -502,18 +519,16 @@ export class DatabaseManager extends DisposableObject {
     progress: ProgressCallback,
     token: vscode.CancellationToken,
     uri: vscode.Uri,
-    options?: DatabaseOptions
   ): Promise<DatabaseItem> {
-
     const contents = await resolveDatabaseContents(uri);
-    const realOptions = options || {};
```

Comment:
What happened to this? Were we never passing in `options` to begin with? Can't that happen when loading a persisted item?

Reply:
No, we were never passing in `options`. We load persisted items from `loadPersistedState()`. So, it is an entirely different path.